proposal.md

```diff
@@ -2,4 +2,6 @@
 
 ### Description of the data
 
+In this project we will be visualizing the gender makeup of job workforces from 255 different jobs between the years 1850 and 2000. Each of the 7650 workforces in our data have 5 associated variables that describe their job (`job`), the binary gender of the workforce (`sex`), the decade their information was collected (`year`), and the proportion of their workforce to that of the decade's total workforce (`perc`). Using this data we will also derive a two new variables for each workforce, the first being the dominant gender group (`gender_dominant_group`) of their jobs total workforce in that decade (only male, male dominant, balanced, female dominant, only female), and the second being the proportion of their workforce to the total workforce of that job in that decade.
```

Comment:
```suggestion
In this project, we will be visualizing the gender makeup of job workforces from 255 different jobs between the years 1850 and 2000. Each of the 7650 workforces in our data has 5 associated variables that state the job name(`job`), the binary gender of the workforce (`sex`), the decade their information was collected (`year`), and the proportion of their workforce to that of the decade's total workforce (`perc`). Using this data, we will also derive two new variables for each workforce. The first is gender group (`gender_dominant_group`), which includes the five options (only male, male dominant, balanced, female dominant, only female), based on the workforce's male-to-female ratio. The second is the proportion of the workforce to the entire job market in that decade.
```

Reply:
@bbaillie My suggestion made a few changes mainly for clarification and Grammarly suggestions from the original.

Reply:
Ill take these into consideration and make some new changed